modules/rostests/apitests/gdi32/GetObject.c

```diff
@@ -716,5 +728,6 @@ START_TEST(GetObject)
     Test_ExtPen(); // not implemented yet in ROS
     Test_MetaDC();
     Test_Region();
+    Test_CursorIcon();
```

Comment:
Why calling the test function two times?

Reply:
When your new test is run in Windows Server 2003 sp2, the results indicate that the correct response is 64 and NOT 32.

![gdi32_GetObject_2K3sp2](https://user-images.githubusercontent.com/32620577/132898504-63ea1595-e5fd-4363-a995-95c58f8afc9f.png)

It appears that this test is not correct to me. Thanks.

Reply:
See also https://devblogs.microsoft.com/oldnewthing/20210819-00/?p=105572

Reply:
I deduce here that there is a big difference between XP sp2 and sp3 (sp3 reports 32)

Reply:
Also, try changing your display settings and the virtual machine software in which you run your given windows version...

Reply:
I deduce here that there is a big difference between XP sp2 and sp3 (sp3 reports 32)

Reply:
Also, please see https://stackoverflow.com/questions/68810645/loadcursor-and-mixed-dpi-multiple-monitors
> When the LoadCursor function is used to load a cursor from a resource, the resulting HCURSOR can be used across different monitors and always appears at the correct size.
> ie: typically:
> on a 96 dpi monitor the 32x32 resource is used,
> on a 192 dpi monitor the 64x64 resource is used.